roofit/roofitcore/src/RooAbsCollection.cxx

```diff
@@ -951,6 +951,8 @@ RooAbsArg * RooAbsCollection::find(const RooAbsArg& arg) const
 
   if (_hashAssistedFind || _list.size() >= _sizeThresholdForMapSearch) {
     if (!_hashAssistedFind || !_hashAssistedFind->isValid()) {
+      if (_hashAssistedFind)
```

Comment:
This `if` can be removed

Reply:
I agree! I just followed what was done in the linked PR. Should I remove the if in both lines, here and in the linked PR?

Reply:
If you wish, I think we can do that in one commit here, yes.

Reply:
done